backend/src/main/java/com/ody/mate/dto/response/MateStatusResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.ody.mate.dto.response;
+
+import com.ody.mate.domain.EtaStatus;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public record MateStatusResponse(
+
+        @Schema(description = "참여자 닉네임",  example = "콜리")
+        String nickname,
+
+        @Schema(description = "참여자 ETA에 따른 상태",  example = "LATE_WARNING")
+        EtaStatus status,
+
+        @Schema(description = "도착지까지 남은 소요시간",  example = "32")
```

Comment:
[제안]

'분' 단위도 명시해주면 어떨까요?

```suggestion
        @Schema(description = "도착지까지 남은 소요시간(분)",  example = "32")
```

Reply:
소요시간이라 되어있으면 헷갈릴 여지가 있을 것 같네요 해당 부분 반영했습니다 👍🏻